### PR TITLE
Multilabel prediction

### PR DESCRIPTION
**Pull request by jwmueller**
Adds tutorial for multilabel tabular prediction.

Related issue: https://github.com/awslabs/autogluon/issues/228

### CODE REVIEW

**Comment by szha** [comment]
Job PR-953-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-953/2/index.html

**Comment by szha** [comment]
Job PR-953-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-953/3/index.html

File: docs/tutorials/index.rst [hunk 1]
```diff
@@ -50,14 +50,31 @@ Tabular Prediction
...
 
    .. card::
       :title: Kaggle Tutorial
       :link: tabular_prediction/tabular-kaggle.html
 
       Using AutoGluon for Kaggle competitions with tabular data.
 
+   .. card::
+      :title: Data Tables Containing Text
+      :link: tabular-multimodal-text-others.html
```
Comment (Innixma) [inline docs/tutorials/index.rst:63]: 403 Forbidden, wrong path

http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-953/3/tutorials/index.html

File: docs/tutorials/index.rst [hunk 2]
```diff
@@ -50,14 +50,31 @@ Tabular Prediction
...
+
+      Modeling data tables with text and numeric/categorical features.
+
+   .. card::
+      :title: Multi-Label Prediction
```
Comment (Innixma) [inline docs/tutorials/index.rst:68]: 403 Forbidden, wrong path

http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-953/3/tutorials/index.html

File: docs/tutorials/index.rst [hunk 3]
```diff
@@ -50,14 +50,31 @@ Tabular Prediction
...
+      :link: tabular-multilabel.html
+
+      How to predict multiple columns in a data table.
+
+   .. card::
+      :title: FAQ
+      :link: tabular-faq.html
```
Comment (Innixma) [inline docs/tutorials/index.rst:75]: 403 Forbidden, wrong path

http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-953/3/tutorials/index.html

File: docs/tutorials/index.rst (hunk 1 above)
Comment (Innixma) [reply docs/tutorials/index.rst:63]: Must prefix with `tabular_prediction/`

File: docs/tutorials/index.rst (hunk 2 above)
Comment (Innixma) [reply docs/tutorials/index.rst:68]: Must prefix with `tabular_prediction/`

File: docs/tutorials/index.rst (hunk 3 above)
Comment (Innixma) [reply docs/tutorials/index.rst:75]: Must prefix with `tabular_prediction/`

File: docs/tutorials/tabular_prediction/tabular-multilabel.md [hunk 4]
```diff
@@ -61,7 +61,7 @@ class MultilabelPredictor():
         eval_metric = None
         for i in range(len(labels)):
             label = labels[i]
-            path_i = self.path + "/Predictor_" + label + "/"
+            path_i = self.path + "Predictor_" + label + "/"
```
Comment (Innixma) [inline docs/tutorials/tabular_prediction/tabular-multilabel.md:64]: try to always use `os.path.sep` , currently this may break on windows.

**Comment by Innixma** [comment]
@jwmueller CI failed

**Comment by szha** [comment]
Job PR-953-7 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-953/7/index.html

**Review by Innixma — APPROVED** [review]
Looks good!

**Pull request merged**